teams/2/assets/css/blog_posts.css

```diff
@@ -1,110 +1,209 @@
-*{
-    font-family: Arial, Helvetica, sans-serif;
-  
-} 
-
-body, ul {
-    margin: 0%;
-    padding: 0%;
-    list-style-type: none;
-    
-}
-header{
+.ut-top-navbar {
+    position: fixed;
+    top: 0;
+    background-color: #f5f5f5;
+    z-index: 22;
     width: 100%;
+    padding: 0 20px;
 }
 
+header {
```

Comment:
do not target elements within your styles, reuse the headers styles that team 1 created